bitnami/redis/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: redis
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 18.4.0
+version: 18.4.3
```

I am not sure about this jump in the versions but, keeping in mind that you  are adding a new feature, I think is worth to bump the minor version:

```suggestion
version: 18.5.0
```